From f777e2a9241c4f384f60c467e226eee85fb5bc98 Mon Sep 17 00:00:00 2001
From: Henrik Fehlauer <rkflx@lab12.net>
Date: Sat, 4 Nov 2017 00:22:09 +0100
Subject: Fix CMake Error in generators/spectre/CMakeLists.txt

Summary:
When porting from `add_library` to `okular_add_generator`, 2f9246ae42ce
missed to remove `MODULE` for spectre, which is already implicitly added
as can be seen [here](https://phabricator.kde.org/source/kcoreaddons/browse/master/KF5CoreAddonsMacros.cmake;83623a46c9862ee37535de93427dec0b0c55e24d$139).

This causes the upcoming CMake 3.10 to print the error `Cannot find
source file`, which can be fixed by removing the superfluous `MODULE`.

BUG: 386176

Test Plan:
No error shown anymore with CMake 3.10.0-rc4. Still works with CMake
3.5.1.

Reviewers: #okular, ngraham

Reviewed By: ngraham

Tags: #okular

Differential Revision: https://phabricator.kde.org/D8612
---
 generators/spectre/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/generators/spectre/CMakeLists.txt b/generators/spectre/CMakeLists.txt
index d808c36..5d6927d 100644
--- a/generators/spectre/CMakeLists.txt
+++ b/generators/spectre/CMakeLists.txt
@@ -24,7 +24,7 @@ ki18n_wrap_ui(okularGenerator_ghostview_SRCS
 kconfig_add_kcfg_files(okularGenerator_ghostview_SRCS conf/gssettings.kcfgc )
 
 
-okular_add_generator(okularGenerator_ghostview MODULE ${okularGenerator_ghostview_SRCS})
+okular_add_generator(okularGenerator_ghostview ${okularGenerator_ghostview_SRCS})
 
 target_link_libraries(okularGenerator_ghostview okularcore ${LIBSPECTRE_LIBRARY} KF5::I18n Qt5::Xml)
 
-- 
cgit v0.11.2
